### DISCUSSION TITLE
图片加载异常

### DISCUSSION THREAD

**Discussion by Tropical8818** [halo-dev/halo#3154] [Q&A]
当按照官方文档的安装设置完成后，附件存储策略不变也就是默认在本地，然后在附件中上传照片后，缩略图显示“加载异常”，且双击打开后也是显示同样的问题。也就是上传的照片无法预览和打开观看。

**Comment by ruibaby** [comment]
你需要检查 `halo.external-url` 参数是否正常设置。

https://docs.halo.run/getting-started/install/docker-compose
